gluten-core/src/main/scala/org/apache/spark/softaffinity/SoftAffinity.scala

```diff
@@ -39,7 +39,7 @@ abstract class Affinity(val manager: AffinityManager) extends LogLevelUtil with
       filePaths: Array[String],
       preferredLocations: Array[String]): Array[String] = {
     if (shouldUseSoftAffinity(filePaths, preferredLocations)) {
-      internalGetHostLocations(filePaths.min)
+      internalGetHostLocations(filePaths(0))
```

Comment:
Is this modification related to this issue?

Reply:
not related